sklearn/metrics/_scorer.py

```diff
@@ -371,7 +372,8 @@ def _passthrough_scorer(estimator, *args, **kwargs):
     return estimator.score(*args, **kwargs)
 
 
-def check_scoring(estimator, scoring=None, allow_none=False):
+@_deprecate_positional_args
+def check_scoring(estimator, *, scoring=None, allow_none=False):
```

Comment:
`*` after scoring, please, since it is patent from the name that scoring should be expected, and I think the order of estimator and scoring will be unambiguous in context.